### DISCUSSION TITLE
Change the checkout header to kept the user as far as possible in the purchase funnel

### DISCUSSION THREAD

**Discussion by ElodieOS** [PrestaShop/hummingbird#818] [Ideas]
Hi everyone! 

We have an improvement and we'd like your opinion. 

We think the header for the checkout flow should be lighter. The navigation should be removed so that the user stays focused and is kept as far as possible in the purchase funnel (without feeling trapped). 

Wdyt?

![image](https://github.com/PrestaShop/hummingbird/assets/113457393/98644c87-9113-47a4-9ed4-b5620e93338d)

**Comment by SharakPL** [comment]
I agree. It's been like that in classic and classic-rocket.

**Comment by Hlavtox** [comment]
I disagree. Removing menu would just hurt accesibility and make it harder for the customer to add something to cart. Why throw branches under his legs?

Also, the customer probably got used to the layout of the website, where the basics like search, menu, cart icon is in the header, and suddenly it changes? Also not good for accesibility.

Yes, it's popular somewhere, but that doesn't make it good, in my opinion.

> without feeling trapped

Right now he is trapped and removing hamburger icon will make him not trapped? Come on. 😄
